packages/frontend/package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "format": "yarn prettier -w"
```

Comment:
If I run this task directly (entering into the folder and running "yarn format") nothing happens.

I guess it's because we are not specifying the sources files? From scaffold-eth: `"prettier": "npx prettier --write . '!(node_module|build)/**/*'"`

Should be similar. But I think we don't need "yarn" nor "npx" and can run prettier directly